src/views/client-dashboard/ClientPayments.vue

```diff
@@ -10,7 +10,34 @@
             :class="`${card.primary ? 'active' : ''} client-payment-card-item`"
             @click.prevent="onSelect(card)"
           >
-            <div class="card-number" v-html="getCardInfo(card)" />
+            <div class="d-flex align-items-center">
+              <img
+                v-if="['mastercard', 'MasterCard'].includes(card.brand)"
+                src="@/assets/svg/mastercard.svg"
+                alt="mastercard"
+                width="30"
+                height="30"
+                class="lazy ml-n1 mr-1"
+              >
+              <img
+                v-else-if="['visa', 'Visa'].includes(card.brand)"
+                src="@/assets/svg/visa.svg"
+                alt="visa"
+                width="30"
+                height="30"
+                class="lazy ml-n1 mr-1"
+              >
+              <img
+                v-else-if="['amex', 'American Express'].includes(card.brand)"
```

Comment:
same here as for mastercard

Reply:
here seems to be another scenario. The options are spelled different.

Reply:
true